app/boards/arm/corneish_zen/corneish_zen.dtsi

```diff
@@ -0,0 +1,128 @@
+/*
+*
+* Copyright (c) 2021 Darryl deHaan
+* SPDX-License-Identifier: MIT
+*
+*/
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+	model = "corneish_zen_v2";
+	compatible = "corneish_zen_v2";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zmk,kscan = &kscan0;
+		zmk,matrix_transform = &default_transform;
+		//zmk,matrix_transform = &five_column_transform;
```

Comment:
Probably better left as a comment in the keymap if you want to leave that as an example in a comment.

Reply:
This transform line is not referenced from the keymap file though... maybe I am not understanding.

Reply:
figured it out... just moved everything over to the keymap